pkg/postgres/rows.go

```diff
@@ -1,17 +1,35 @@
 package postgres
 
 import (
+	"context"
+
 	"github.com/jackc/pgx/v4"
 )
 
+// Row wraps pgx.Row
+type Row struct {
+	pgx.Row
+	cancelFunc context.CancelFunc
+}
+
+// Scan wraps pgx.Row Scan
+func (r *Row) Scan(dest ...interface{}) error {
+	defer r.cancelFunc()
```

Comment:
does this cancel the context the first time we call scan?  I think that is OK because once we are calling scan the query is complete and we are processing the results.  I just want to make sure that is what it is doing and that is the intent.

Reply:
this is just the raw row which does not have next. So the only interface is Scan() which means close will always be called inside scan even if the context has expired